From 46251ff000f424491e14bf866ffe6b62f379baae Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 15 Apr 2014 10:22:56 +0800
Subject: [PATCH 3/4] ENGR00308369 enable camera function for 6sx sdb board.

when there are GPU3D and VPU module, use new Camera2.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 7e97fbf..7fcec47 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,4 +1,5 @@
 ifeq ($(HAVE_FSL_IMX_GPU3D),true)
+ifeq ($(BOARD_HAVE_VPU),true)
 LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -41,3 +42,4 @@ include $(BUILD_PACKAGE)
 
 include $(call all-makefiles-under, $(LOCAL_PATH))
 endif
+endif
-- 
1.8.0

